`HTTP 404`: Cannot connect Nextcloud server to app

**Describe the bug**
Cannot add account to app.


**To Reproduce**
Steps to reproduce the behavior:
1. Open the app
2. Connect Nextcloud account
3. Error occurs


**Expected behavior**
To be able to connect to my tasks via the app.

**Screenshots**
![Screenshot_20230907-142211.png](https://github.com/nextcloud/notes-android/assets/8862291/75f331e4-60fd-466b-a968-4241a624f02f)

**Smartphone (please complete the following information):**
 - Nextcloud Notes-Version (android app): 4.0.0
 - F-Droid or Play Store: play
 - Android-Version: 5.10.149-android12-9-00003-g230976ad916b-ab9958333(9d464-2425b)
 - Device: moto g73 5G


**Server**
 - Nextcloud version: 26.0.4
 - Nextcloud Notes version (server app): 3.7.2

**Error code**
```
App Version: 4.0.0
App Version Code: 40000090
App Flavor: play

Files App Version Code: 30250090

---

OS Version: 5.10.149-android12-9-00003-g230976ad916b-ab9958333(9d464-2425b)
OS API Level: 33
Device: devonf
Manufacturer: motorola
Model (and Product): moto g73 5G (devonf_g_syse)

---

com.nextcloud.android.sso.exceptions.NextcloudHttpRequestFailedException: HTTP forespørsel feilet med status kode: 404
	at com.nextcloud.android.sso.api.AidlNetworkRequest.performNetworkRequestV2(AidlNetworkRequest.java:188)
	at com.nextcloud.android.sso.api.NextcloudAPI.performNetworkRequestV2(NextcloudAPI.java:199)
	at com.nextcloud.android.sso.api.NextcloudAPI.lambda$performRequestObservableV2$1$com-nextcloud-android-sso-api-NextcloudAPI(NextcloudAPI.java:129)
	at com.nextcloud.android.sso.api.NextcloudAPI$$ExternalSyntheticLambda0.subscribe(Unknown Source:6)
	at io.reactivex.internal.operators.observable.ObservableFromPublisher.subscribeActual(ObservableFromPublisher.java:31)
	at io.reactivex.Observable.subscribe(Observable.java:12284)
	at io.reactivex.internal.operators.observable.ObservableMap.subscribeActual(ObservableMap.java:32)
	at io.reactivex.Observable.subscribe(Observable.java:12284)
	at io.reactivex.internal.operators.observable.ObservableSingleMaybe.subscribeActual(ObservableSingleMaybe.java:31)
	at io.reactivex.Maybe.subscribe(Maybe.java:4290)
	at io.reactivex.Maybe.blockingGet(Maybe.java:2320)
	at io.reactivex.Observable.blockingSingle(Observable.java:5381)
	at it.niedermann.owncloud.notes.persistence.NotesImportTask.lambda$importNotes$1$it-niedermann-owncloud-notes-persistence-NotesImportTask(NotesImportTask.java:57)
	at it.niedermann.owncloud.notes.persistence.NotesImportTask$$ExternalSyntheticLambda1.run(Unknown Source:6)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:463)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: java.lang.IllegalStateException: <!DOCTYPE html>
<html class="ng-csp" data-placeholder-focus="false" lang="nb" data-locale="nn" translate="no" >
	<head
 data-requesttoken="">
		<meta charset="utf-8">
		<title>
			Munin		</title>
		<meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
				<meta name="apple-itunes-app" content="app-id=1125420102">
				<meta name="theme-color" content="#8855a8">
		<link rel="icon" href="/core/img/favicon.ico">
		<link rel="apple-touch-icon" href="/core/img/favicon-touch.png">
		<link rel="mask-icon" sizes="any" href="/core/img/favicon-mask.svg" color="#8855a8">
		<link rel="manifest" href="/index.php/apps/theming/manifest?v=7df4fde3">
		<link rel="stylesheet" href="/core/css/server.css?v=94256dd2-9">
<link rel="stylesheet" href="/apps/files_rightclick/css/app.css?v=35c36e89-9">
<link rel="stylesheet" href="/extra-apps/spreed/css/unified-search.css?v=e2114d19-9">
<link rel="stylesheet" href="/apps/theming/css/default.css?v=9db53190-9">
<link rel="stylesheet" href="/core/css/guest.css?v=94256dd2-9">
		<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/dist/core-common.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/dist/core-main.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/core/l10n/nb.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/dist/core-files_fileinfo.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/dist/core-files_client.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/index.php/js/core/merged-template-prepend.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/apps/files_sharing/l10n/nb.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/dist/files_sharing-main.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/apps/files_rightclick/l10n/nb.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/apps/files_rightclick/js/script.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/apps/files_rightclick/js/files.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/apps/firstrunwizard/l10n/nb.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/apps/firstrunwizard/js/about.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/apps/theming/l10n/nb.js?v=94256dd2-9"></script>
<script nonce="YzR0YVlRRS9UdHRGSU84cFVVR2kzeHU3VnNmMmYwMnBSdThmKzFhbzh4WT06Q3Y0eU5sQUpLNm9HVVo1ZEppenJ1R3JkRUltZENnYmdOSVJMZ2lmK3NuUT0=" defer src="/apps/theming/js/theming.js?v=94256dd2-9"></script>
		<link rel="stylesheet" media="" href="/index.php/apps/theming/theme/default.css?plain=1&amp;v=7df4fde3" class="theme"/><link rel="stylesheet" media="(prefers-color-scheme: light)" href="/index.php/apps/theming/theme/light.css?plain=1&amp;v=7df4fde3" class="theme"/><link rel="stylesheet" media="(prefers-color-scheme: dark)" href="/index.php/apps/theming/theme/dark.css?plain=1&amp;v=7df4fde3" class="theme"/><link rel="stylesheet" media="(prefers-contrast: more)" href="/index.php/apps/theming/theme/light-highcontrast.css?plain=1&amp;v=7df4fde3" class="theme"/><link rel="stylesheet" media="(prefers-color-scheme: dark) and (prefers-contrast: more)" href="/index.php/apps/theming/theme/dark-highcontrast.css?plain=1&amp;v=7df4fde3" class="theme"/><link rel="stylesheet" media="" href="/index.php/apps/theming/theme/light.css?plain=0&amp;v=7df4fde3" class="theme"/><link rel="stylesheet" media="" href="/index.php/apps/theming/theme/dark.css?plain=0&amp;v=7df4fde3" class="theme"/><link rel="stylesheet" media="" href="/index.php/apps/theming/theme/light-highcontrast.css?plain=0&amp;v=7df4fde3" class="theme"/><link rel="stylesheet" media="" href="/index.php/apps/theming/theme/dark-highcontrast.css?plain=0&amp;v=7df4fde3" class="theme"/><link rel="stylesheet" media="" href="/index.php/apps/theming/theme/opendyslexic.css?plain=0&amp;v=7df4fde3" class="theme"/><meta name="robots" content="noindex, nofollow"/>	</head>
	<body id="body-login">
		<noscript>
	<div id="nojavascript">
		<div>
			Denne appen krever JavaScript for å fungere korrekt. <a href="https://www.enable-javascript.com/" target="_blank" rel="noreferrer noopener">Aktiver JavaScript</a> og last siden på nytt.		</div>
	</div>
</noscript>
					<input type="hidden" id="initial-state-theming-backgroundImage" value="ImRlZmF1bHQi">
					<input type="hidden" id="initial-state-theming-backgroundColor" value="IiMwMDgyYzki">
					<input type="hidden" id="initial-state-theming-themingDefaultBackground" value="ImJhY2tncm91bmRDb2xvciI=">
					<input type="hidden" id="initial-state-theming-defaultShippedBackground" value="ImthbWlsLXBvcmVtYmluc2tpLWNsb3Vkcy5qcGci">
					<input type="hidden" id="initial-state-theming-shippedBackgrounds" value="eyJoYW5uYWgtbWFjbGVhbi1zb2Z0LWZsb3JhbC5qcGciOnsiYXR0cmlidXRpb24iOiJTb2Z0IGZsb3JhbCAoSGFubmFoIE1hY0xlYW4sIENDMCkiLCJhdHRyaWJ1dGlvbl91cmwiOiJodHRwczpcL1wvc3RvY2tzbmFwLmlvXC9waG90b1wvc29mdC1mbG9yYWwtWE9ZV0NDVzVQQSIsInRoZW1pbmciOiJkYXJrIiwicHJpbWFyeV9jb2xvciI6IiM5ZjY1MmYifSwidGVkLW1vcmF2ZWMtbW9ybmluZy1mb2cuanBnIjp7ImF0dHJpYnV0aW9uIjoiTW9ybmluZyBmb2cgKFRlZCBNb3JhdmVjLCBQdWJsaWMgRG9tYWluKSIsImF0dHJpYnV0aW9uX3VybCI6Imh0dHBzOlwvXC9mbGlja3IuY29tXC9waG90b3NcL3Rtb3JhdmVjXC81MjM5MjQxMDI2MSIsInRoZW1pbmciOiJkYXJrIiwicHJpbWFyeV9jb2xvciI6IiMxMTRjM2IifSwic3RlZmFudXMtbWFydGFudG8tc2V0eW8taHVzb2RvLXVuZGVyd2F0ZXItb2NlYW4uanBnIjp7ImF0dHJpYnV0aW9uIjoiVW5kZXJ3YXRlciBvY2VhbiAoU3RlZmFudXMgTWFydGFudG8gU2V0eW8gSHVzb2RvLCBDQzApIiwiYXR0cmlidXRpb25fdXJsIjoiaHR0cHM6XC9cL3N0b2Nrc25hcC5pb1wvcGhvdG9cL3VuZGVyd2F0ZXItb2NlYW4tVEpBOUxCSDRXUyIsInByaW1hcnlfY29sb3IiOiIjMDQ1NzdlIn0sInpvbHRhbi12b3Jvcy1yaHl0aG0tYW5kLWJsdWVzLmpwZyI6eyJhdHRyaWJ1dGlvbiI6IlJoeXRobSBhbmQgYmx1ZXMgKFpvbHRcdTAwZTFuIFZcdTAwZjZyXHUwMGY2cywgQ0MgQlkpIiwiYXR0cmlidXRpb25fdXJsIjoiaHR0cHM6XC9cL2ZsaWNrci5jb21cL3Bob3Rvc1wvdjkyM3pcLzUxNjM0NDA5Mjg5XC8iLCJwcmltYXJ5X2NvbG9yIjoiIzFjMjQzYyJ9LCJhbmF0b2x5LW1pa2hhbHRzb3YtYnV0dGVyZmx5LXdpbmctc2NhbGUuanBnIjp7ImF0dHJpYnV0aW9uIjoiQnV0dGVyZmx5IHdpbmcgc2NhbGUgKEFuYXRvbHkgTWlraGFsdHNvdiwgQ0MgQlktU0EpIiwiYXR0cmlidXRpb25fdXJsIjoiaHR0cHM6XC9cL2NvbW1vbnMud2lraW1lZGlhLm9yZ1wvd2lraVwvRmlsZTolRDAlQTclRDAlQjUlRDElODglRDElODMlRDAlQjklRDAlQkElRDAlQjhfJUQwJUJBJUQxJTgwJUQxJThCJUQwJUJCJUQwJUIwXyVEMCVCMSVEMCVCMCVEMCVCMSVEMCVCRSVEMSU4NyVEMCVCQSVEMCVCOC5qcGciLCJwcmltYXJ5X2NvbG9yIjoiI2E1M2MxNyJ9LCJiZXJuaWUtY2V0b25pYS1hdXJhdGEtdGFrZS1vZmYtY29tcG9zaXRpb24uanBnIjp7ImF0dHJpYnV0aW9uIjoiQ2V0b25pYSBhdXJhdGEgdGFrZSBvZmYgY29tcG9zaXRpb24gKEJlcm5pZSwgUHVibGljIERvbWFpbikiLCJhdHRyaWJ1dGlvbl91cmwiOiJodHRwczpcL1wvY29tbW9ucy53aWtpbWVkaWEub3JnXC93aWtpXC9GaWxlOkNldG9uaWFfYXVyYXRhX3Rha2Vfb2ZmX2NvbXBvc2l0aW9uXzA1MTcyMDA5LmpwZyIsInRoZW1pbmciOiJkYXJrIiwicHJpbWFyeV9jb2xvciI6IiM1NjYzM2QifSwiZGVqYW4ta3JzbWFub3ZpYy1yaWJiZWQtcmVkLW1ldGFsLmpwZyI6eyJhdHRyaWJ1dGlvbiI6IlJpYmJlZCByZWQgbWV0YWwgKERlamFuIEtyc21hbm92aWMsIENDIEJZKSIsImF0dHJpYnV0aW9uX3VybCI6Imh0dHBzOlwvXC93d3cuZmxpY2tyLmNvbVwvcGhvdG9zXC9kZWphbmtyc21hbm92aWNcLzQyOTcxNDU2Nzc0XC8iLCJwcmltYXJ5X2NvbG9yIjoiIzljNDIzNiJ9LCJlZHVhcmRvLW5ldmVzLXBlZHJhLWF6dWwuanBnIjp7ImF0dHJpYnV0aW9uIjoiUGVkcmEgYXp1bCBtaWxreSB3YXkgKEVkdWFyZG8gTmV2ZXMsIENDIEJZLVNBKSIsImF0dHJpYnV0aW9uX3VybCI6Imh0dHBzOlwvXC9jb21tb25zLndpa2ltZWRpYS5vcmdcL3dpa2lcL0ZpbGU6UGVkcmFfQXp1bF9NaWxreV9XYXkuanBnIiwicHJpbWFyeV9jb2xvciI6IiM0ZjYwNzEifSwiZXVyb3BlYW4tc3BhY2UtYWdlbmN5LWJhcmVudHMtYmxvb20uanBnIjp7ImF0dHJpYnV0aW9uIjoiQmFyZW50cyBibG9vbSAoRXVyb3BlYW4gU3BhY2UgQWdlbmN5LCBDQyBCWS1TQSkiLCJhdHRyaWJ1dGlvbl91cmwiOiJodHRwczpcL1wvd3d3LmVzYS5pbnRcL0VTQV9NdWx0aW1lZGlhXC9JbWFnZXNcLzIwMTZcLzA4XC9CYXJlbnRzX2Jsb29tIiwicHJpbWFyeV9jb2xvciI6IiMzOTY0NzUifSwiaGFubmVzLWZyaXR6LWZsaXBwaXR5LWZsb3BwaXR5LmpwZyI6eyJhdHRyaWJ1dGlvbiI6IkZsaXBwaXR5IGZsb3BwaXR5IChIYW5uZXMgRnJpdHosIENDIEJZLVNBKSIsImF0dHJpYnV0aW9uX3VybCI6Imh0dHA6XC9cL2hhbm5lcy5waG90b3NcL2ZsaXBwaXR5LWZsb3BwaXR5IiwicHJpbWFyeV9jb2xvciI6IiM5ODQxNWEifSwiaGFubmVzLWZyaXR6LXJvdWxldHRlLmpwZyI6eyJhdHRyaWJ1dGlvbiI6IlJvdWxldHRlIChIYW5uZXMgRnJpdHosIENDIEJZLVNBKSIsImF0dHJpYnV0aW9uX3VybCI6Imh0dHA6XC9cL2hhbm5lcy5waG90b3NcL3JvdWxldHRlIiwicHJpbWFyeV9jb2xvciI6IiM4NDUzMzQifSwiaGFubmVzLWZyaXR6LXNlYS1zcHJheS5qcGciOnsiYXR0cmlidXRpb24iOiJTZWEgc3ByYXkgKEhhbm5lcyBGcml0eiwgQ0MgQlktU0EpIiwiYXR0cmlidXRpb25fdXJsIjoiaHR0cDpcL1wvaGFubmVzLnBob3Rvc1wvc2VhLXNwcmF5IiwicHJpbWFyeV9jb2xvciI6IiM0ZjYwNzEifSwia2FtaWwtcG9yZW1iaW5za2ktY2xvdWRzLmpwZyI6eyJhdHRyaWJ1dGlvbiI6IkNsb3VkcyAoS2FtaWwgUG9yZW1iaVx1MDE0NHNraSwgQ0MgQlktU0EpIiwiYXR0cmlidXRpb25fdXJsIjoiaHR0cHM6XC9cL3d3dy5mbGlja3IuY29tXC9waG90b3NcL3Bhc3pjemFrMDAwXC84NzE1ODUxNTIxXC8iLCJwcmltYXJ5X2NvbG9yIjoiIzAwODJjOSJ9LCJiZXJuYXJkLXNwcmFnZy1uZXctemVhbGFuZC1mZXJuLmpwZyI6eyJhdHRyaWJ1dGlvbiI6Ik5ldyB6ZWFsYW5kIGZlcm4gKEJlcm5hcmQgU3ByYWdnLCBDQzApIiwiYXR0cmlidXRpb25fdXJsIjoiaHR0cHM6XC9cL2NvbW1vbnMud2lraW1lZGlhLm9yZ1wvd2lraVwvRmlsZTpOWl9GZXJuLihCbGVjaG51bV9jaGFtYmVyc2lpKV8oMTEyNjM1MzQ5MzYpLmpwZyIsInByaW1hcnlfY29sb3IiOiIjMzE2YjI2In0sInJhd3BpeGVsLXBpbmstdGFwaW9jYS1idWJibGVzLmpwZyI6eyJhdHRyaWJ1dGlvbiI6IlBpbmsgdGFwaW9jYSBidWJibGVzIChSYXdwaXhlbCwgQ0MgQlkpIiwiYXR0cmlidXRpb25fdXJsIjoiaHR0cHM6XC9cL3d3dy5mbGlja3IuY29tXC9waG90b3NcL2J5cmF3cGl4ZWxcLzI3NjY1MTQwMjk4XC9pblwvcGhvdG9zdHJlYW1cLyIsInRoZW1pbmciOiJkYXJrIiwicHJpbWFyeV9jb2xvciI6IiM3YjRlN2UifSwibmFzYS13YXhpbmctY3Jlc2NlbnQtbW9vbi5qcGciOnsiYXR0cmlidXRpb24iOiJXYXhpbmcgY3Jlc2NlbnQgbW9vbiAoTkFTQSwgUHVibGljIERvbWFpbikiLCJhdHRyaWJ1dGlvbl91cmwiOiJodHRwczpcL1wvd3d3Lm5hc2EuZ292XC9pbWFnZS1mZWF0dXJlXC9hLXdheGluZy1jcmVzY2VudC1tb29uIiwicHJpbWFyeV9jb2xvciI6IiMwMDVhYzEifSwidG9tbXktY2hhdS1hbHJlYWR5LmpwZyI6eyJhdHRyaWJ1dGlvbiI6IkNpdHlzY2FwZSAoVG9tbXkgQ2hhdSwgQ0MgQlkpIiwiYXR0cmlidXRpb25fdXJsIjoiaHR0cHM6XC9cL3d3dy5mbGlja3IuY29tXC9waG90b3NcLzkwOTc1NjkzQE4wNVwvMTY5MTA5OTkzNjgiLCJwcmltYXJ5X2NvbG9yIjoiIzZhMmFmNCJ9LCJ0b21teS1jaGF1LWxpb24tcm9jay1oaWxsLmpwZyI6eyJhdHRyaWJ1dGlvbiI6Ikxpb24gcm9jayBoaWxsIChUb21teSBDaGF1LCBDQyBCWSkiLCJhdHRyaWJ1dGlvbl91cmwiOiJodHRwczpcL1wvd3d3LmZsaWNrci5jb21cL3Bob3Rvc1wvOTA5NzU2OTNATjA1XC8xNzEzNjQ0MDI0NiIsInRoZW1pbmciOiJkYXJrIiwicHJpbWFyeV9jb2xvciI6IiM3ZjRmNzAifSwibGFsaS1tYXNyaWVyYS15ZWxsb3ctYnJpY2tzLmpwZyI6eyJhdHRyaWJ1dGlvbiI6IlllbGxvdyBicmlja3MgKExhbGkgTWFzcmllcmEsIENDIEJZKSIsImF0dHJpYnV0aW9uX3VybCI6Imh0dHBzOlwvXC93d3cuZmxpY2tyLmNvbVwvcGhvdG9zXC92aXN1YWxwYW5pY1wvMzk4MjQ2NDQ0NyIsInRoZW1pbmciOiJkYXJrIiwicHJpbWFyeV9jb2xvciI6IiM3ZjU3MDAifX0=">
					<input type="hidden" id="initial-state-theming-data" value="eyJuYW1lIjoiTXVuaW4iLCJ1cmwiOiJodHRwczpcL1wvbmV4dGNsb3VkLm9yZW1tZS5ub1wvIiwic2xvZ2FuIjoiIiwiY29sb3IiOiIjODg1NWE4IiwiZGVmYXVsdENvbG9yIjoiIzg4NTVhOCIsImltcHJpbnRVcmwiOiIiLCJwcml2YWN5VXJsIjoiIiwiaW52ZXJ0ZWQiOmZhbHNlLCJjYWNoZUJ1c3RlciI6IjdkZjRmZGUzIiwiZW5hYmxlZFRoZW1lcyI6W119">
					<input type="hidden" id="initial-state-core-versionHash" value="Ijk0MjU2ZGQyIg==">
					<input type="hidden" id="initial-state-comments-maxAutoCompleteResults" value="MTA=">
				<div class="wrapper">
			<div class="v-align">
									<header role="banner">
						<div id="header">
							<div class="logo"></div>
						</div>
					</header>
								<main>
					<h1 class="hidden-visually">
						Munin					</h1>
						<div class="body-login-container update">
		<div class="icon-big icon-search"></div>
		<h2>Siden ble ikke funnet</h2>
		<p class="infogroup">Siden ble ikke funnet på serveren.</p>
		<p><a class="button primary" href="/index.php">
			Tilbake til Munin		</a></p>
	</div>
				</main>
			</div>
		</div>
		<footer role="contentinfo">
			<p class="info">
				<a href="https://nextcloud.oremme.no/" target="_blank" rel="noreferrer noopener" class="entity-name">Munin</a>			</p>
		</footer>
	</body>
</html>

	at com.nextcloud.android.sso.InputStreamBinder.processRequestV2(InputStreamBinder.java:454)
	at com.nextcloud.android.sso.InputStreamBinder.performNextcloudRequestAndBodyStreamV2(InputStreamBinder.java:127)
	at com.nextcloud.android.sso.InputStreamBinder.performNextcloudRequestV2(InputStreamBinder.java:110)
	at com.nextcloud.android.sso.aidl.IInputStreamService$Stub.onTransact(IInputStreamService.java:158)
	at android.os.Binder.execTransactInternal(Binder.java:1294)
	at android.os.Binder.execTransact(Binder.java:1253)

```

I'm encountering what seems to be the same issue.
I'm running an up-to-date Nextcloud and the notes app was installed from google store.

> ```
> App Version: 4.1.0
> App Version Code: 40010090
> App Flavor: play
> 
> Files App Version Code: 30260090 (PROD)
> 
> ---
> 
> OS Version: 4.4.146+(v2B3M-0)
> OS API Level: 28
> Device: Venice
> Manufacturer: TCL
> Model (and Product): 5048U_EEA (5048U_EEA)
> 
> ---
> 
> com.nextcloud.android.sso.exceptions.NextcloudHttpRequestFailedException: o pedido HTTP falhou com código de estado HTTP: 404
> 	at com.nextcloud.android.sso.api.AidlNetworkRequest.performNetworkRequestV2(AidlNetworkRequest.java:197)
> 	at com.nextcloud.android.sso.api.NextcloudAPI.performNetworkRequestV2(NextcloudAPI.java:180)
> 	at com.nextcloud.android.sso.api.NextcloudAPI.lambda$performRequestObservableV2$0$com-nextcloud-android-sso-api-NextcloudAPI(NextcloudAPI.java:122)
> 	at com.nextcloud.android.sso.api.NextcloudAPI$$ExternalSyntheticLambda0.subscribe(Unknown Source:6)
> 	at io.reactivex.internal.operators.observable.ObservableFromPublisher.subscribeActual(ObservableFromPublisher.java:31)
> 	at io.reactivex.Observable.subscribe(Observable.java:12284)
> 	at io.reactivex.internal.operators.observable.ObservableMap.subscribeActual(ObservableMap.java:32)
> 	at io.reactivex.Observable.subscribe(Observable.java:12284)
> 	at io.reactivex.internal.operators.observable.ObservableSingleMaybe.subscribeActual(ObservableSingleMaybe.java:31)
> 	at io.reactivex.Maybe.subscribe(Maybe.java:4290)
> 	at io.reactivex.Maybe.blockingGet(Maybe.java:2320)
> 	at io.reactivex.Observable.blockingSingle(Observable.java:5381)
> 	at it.niedermann.owncloud.notes.persistence.NotesImportTask.lambda$importNotes$1$it-niedermann-owncloud-notes-persistence-NotesImportTask(NotesImportTask.java:57)
> 	at it.niedermann.owncloud.notes.persistence.NotesImportTask$$ExternalSyntheticLambda1.run(Unknown Source:6)
> 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:458)
> 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
> 	at java.lang.Thread.run(Thread.java:764)
> Caused by: java.lang.IllegalStateException: <!DOCTYPE html>
> <html class="ng-csp" data-placeholder-focus="false" lang="pt-PT" data-locale="pt" translate="no" >
>     <SNIPPED>
> </html>
> 
> 	at com.nextcloud.android.sso.InputStreamBinder.processRequestV2(InputStreamBinder.java:454)
> 	at com.nextcloud.android.sso.InputStreamBinder.performNextcloudRequestAndBodyStreamV2(InputStreamBinder.java:127)
> 	at com.nextcloud.android.sso.InputStreamBinder.performNextcloudRequestV2(InputStreamBinder.java:110)
> 	at com.nextcloud.android.sso.aidl.IInputStreamService$Stub.onTransact(IInputStreamService.java:110)
> 	at android.os.Binder.execTransact(Binder.java:731)
> 
> ```

Same issue here.

```
App Version: 4.1.0
App Version Code: 40010090
App Flavor: play

Files App Version Code: 30260090 (PROD)

---

OS Version: 4.19.157-perf+(058002A010012603718185433)
OS API Level: 31
Device: XQ-AS52
Manufacturer: Sony
Model (and Product): XQ-AS52 (XQ-AS52_EEA)

---

com.nextcloud.android.sso.exceptions.NextcloudHttpRequestFailedException: HTTP request failed with HTTP status-code: 502
	at com.nextcloud.android.sso.api.AidlNetworkRequest.performNetworkRequestV2(AidlNetworkRequest.java:197)
	at com.nextcloud.android.sso.api.NextcloudAPI.performNetworkRequestV2(NextcloudAPI.java:180)
	at com.nextcloud.android.sso.api.NextcloudAPI.lambda$performRequestObservableV2$0$com-nextcloud-android-sso-api-NextcloudAPI(NextcloudAPI.java:122)
	at com.nextcloud.android.sso.api.NextcloudAPI$$ExternalSyntheticLambda0.subscribe(Unknown Source:6)
	at io.reactivex.internal.operators.observable.ObservableFromPublisher.subscribeActual(ObservableFromPublisher.java:31)
	at io.reactivex.Observable.subscribe(Observable.java:12284)
	at io.reactivex.internal.operators.observable.ObservableSingleMaybe.subscribeActual(ObservableSingleMaybe.java:31)
	at io.reactivex.Maybe.subscribe(Maybe.java:4290)
	at io.reactivex.Maybe.blockingGet(Maybe.java:2320)
	at io.reactivex.Observable.blockingSingle(Observable.java:5381)
	at it.niedermann.owncloud.notes.persistence.CapabilitiesClient.getCapabilities(CapabilitiesClient.java:32)
	at it.niedermann.owncloud.notes.importaccount.ImportAccountActivity.lambda$onActivityResult$5$it-niedermann-owncloud-notes-importaccount-ImportAccountActivity(ImportAccountActivity.java:96)
	at it.niedermann.owncloud.notes.importaccount.ImportAccountActivity$$ExternalSyntheticLambda2.run(Unknown Source:4)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:487)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: java.lang.IllegalStateException: <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
</body>
</html>

	at com.nextcloud.android.sso.InputStreamBinder.processRequestV2(InputStreamBinder.java:454)
	at com.nextcloud.android.sso.InputStreamBinder.performNextcloudRequestAndBodyStreamV2(InputStreamBinder.java:127)
	at com.nextcloud.android.sso.InputStreamBinder.performNextcloudRequestV2(InputStreamBinder.java:110)
	at com.nextcloud.android.sso.aidl.IInputStreamService$Stub.onTransact(IInputStreamService.java:110)
	at android.os.Binder.execTransactInternal(Binder.java:1179)
	at android.os.Binder.execTransact(Binder.java:1143)

```

--- a/app/src/main/java/it/niedermann/owncloud/notes/exception/tips/TipsAdapter.java
+++ b/app/src/main/java/it/niedermann/owncloud/notes/exception/tips/TipsAdapter.java
@@ -96,6 +96,10 @@ public class TipsAdapter extends RecyclerView.Adapter<TipsViewHolder> {
                         add(R.string.error_dialog_server_app_enabled);
                         add(R.string.error_dialog_redirect);
                     }
+                    case 404 -> {
+                        add(R.string.error_dialog_server_app_enabled);
+                        add(R.string.error_dialog_check_server);
+                    }
                     case 500 -> add(R.string.error_dialog_check_server_logs);
                     case 503 -> add(R.string.error_dialog_check_maintenance);
                     case 507 -> add(R.string.error_dialog_insufficient_storage);

